src/rules/at-forward-before-use/index.js

```diff
@@ -0,0 +1,40 @@
+import { utils } from "stylelint";
+import { namespace } from "../../utils";
+
+export const ruleName = namespace("at-forward-before-use");
+
+export const messages = utils.ruleMessages(ruleName, {
+  rejected: "Unexpected @use at-rule before @forward at-rule"
+});
+
+function rule(primary) {
+  return (root, result) => {
+    const validOptions = utils.validateOptions(result, ruleName, {
+      actual: primary
+    });
+
+    if (!validOptions) {
+      return;
+    }
+
+    const useRules = [];
+
+    root.walkAtRules(/forward|use/, decl => {
+      if (
+        decl.name === "forward" &&
+        useRules.includes(decl.params.replace(/"/g, ""))
```

Comment:
Won't this fail for rules with clauses (`show`/`hide`/`as`/`with`), or rules that use `'` as their quote characters?

Does PostCSS provide a way to run an expression-level parser over the params, so you can consume a string according to precise CSS syntax?

Reply:
@rambleraptor please add more unit tests if there are doubts that the code path does not work in all cases.

Reply:
ping @rambleraptor